core/vm/opcodes.go

```diff
@@ -287,6 +288,7 @@ var opCodeToString = map[OpCode]string{
 	SELFBALANCE: "SELFBALANCE",
 	BASEFEE:     "BASEFEE",
 	BLOBHASH:    "BLOBHASH",
+	BLOBFEE:     "BLOBFEE",
```

Comment:
Just to align on this, EIP specifies `BLOBBASEFEE` as a name. `BLOBFEE` is maybe better naming